### ISSUE TITLE
Cookies are required

### ISSUE THREAD

**Issue by danielzgtg** [jplatte/caniuse.rs#4]
I can't load the website with cookies turned off. It seems that it is trying to use LocalStorage, which requires cookies:

```
Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at https://caniuse.rs/caniuse_rs.js:604:17
    at https://caniuse.rs/caniuse_rs.js:623:2
```

I am using `Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36`.

I have cookies off by default to protect my privacy. It is an inconvenience to whitelist this site, and I would prefer to keep them off anyway.

**Comment by jplatte** [comment]
Oh, I didn't know trying to use localStorage would throw an error if access isn't allowed. Will fix soon.

**Comment by jplatte** [comment]
Should be fixed by 888d9badd2f45762e65b99188d33372ca19dc4cf, let me know if you're still experiencing problems.

**Comment by danielzgtg** [comment]
This works. Thanks!
